ballista/rust/core/proto/ballista.proto

```diff
@@ -503,6 +504,41 @@ message ColumnStats {
   uint32 distinct_count = 4;
 }
 
+message OperatorMetricsSet {
+  repeated OperatorMetric metrics = 1;
+}
+
+
+message NamedCount {
+  string name = 1;
+  uint64 value = 2;
+}
+
+message NamedGauge {
+  string name = 1;
+  uint64 value = 2;
```

Comment:
From the metric system, the Gauge could be negative.
INT64 is better.

Reply:
Good point.

Reply:
@andygrove @alamb 
Can we collation two side?
The `Gauge` use the signed integer to store the value.
https://github.com/apache/arrow-datafusion/pull/1682

Reply:
I would prefer to handle the type alignment in another PR.

Reply:
I think changing `Gauge` in datafusion to use `AtomicI64` rather than `AtomicUsize` would make sense to me